src/ontology/mondo-ingest.Makefile

```diff
@@ -436,11 +436,13 @@ $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-so
 		--rejects $(MAPPINGSDIR)/rejected-mappings.tsv \
 		-o $@
 
+.PHONY: lexical-matches
 lexical-matches: $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv
```

Comment:
## Missing `.PHONY`
_General update: Unrelated to the bug fix at hand, but related to the lexmatch pipeline._